docs/INSTALL.md

```diff
@@ -28,22 +28,15 @@ cd dcos-installer
 Run this command below to have Terraform initialized from this repository. There is **no git clone of this repo required** as Terraform performs this for you.
```

Comment:
Use the following command to initialize Terraform from this repository. There is **no git clone of this repo required** as Terraform performs this for you.